### PR TITLE
Dist clean

### LINKED ISSUE #313: Cleanup dist/ext folder
Mathjax is still into dist/ext folder

### PR DESCRIPTION
**Pull request by tiziodcaio**
Fixes #313

### CODE REVIEW

File: electron/gameWindow.js [hunk 1]
```diff
@@ -37,6 +37,7 @@ async function createWindow(killall) {
     minHeight: 400,
     webPreferences: {
       nativeWindowOpen: true,
+      nodeIntegration: true,
```
Comment (d0sboots) [inline electron/gameWindow.js]: This is a security hole... why are you enabling it?

Comment (tiziodcaio) [reply electron/gameWindow.js]: Yes sorry, I've seen it on stackoverflow like a fix to a problem which appeared... 😐 But now it's working without this security hole!

**Comment by Snarling** [comment]
Those assets are by default fetched from a CDN at runtime, but that doesn't work for us because the game can be played offline. That's why they're bundled with the app in dist/ext.

Does monaco-editor-webpack-plugin create a fully packaged version of monaco? If so we may be able to implement that part, but we will still need to bundle MathJax.

**Comment by tiziodcaio** [comment]
I'm studying the question... Because mathjax is downloading from the CDN for the mathjaxcomponent (and there's an issue opened for that...)

The monaco-editor codebase needs more analysis, because I didn't find out where it calls the cdn and why, if it's bundled it shouldn't, right?

**Pull request closed without merging**